README.md

```diff
@@ -1,4 +1,262 @@
+[Before creating an RFC]: #before-creating-an-rfc
+
+A hastily-proposed RFC can hurt its chances of acceptance. Low quality
+proposals, proposals for previously-rejected features, or those that don't fit
+into the near-term roadmap, may be quickly rejected, which can be demotivating
+for the unprepared contributor. Laying some groundwork ahead of the RFC can
+make the process smoother.
+
+Although there is no single way to prepare for submitting an RFC, it is
+generally a good idea to pursue feedback from other project developers
+beforehand, to ascertain that the RFC may be desirable; having a consistent
+impact on the project requires concerted effort toward consensus-building.
+
+The most common preparations for writing and submitting an RFC include talking
+the idea over on the #ipfs irc channel, filing and discussing ideas on the
+[RFC issue tracker], and occasionally posting "pre-RFCs" on the
+[ipfs discussion forum](https://discuss.ipfs.io) for early review.
+
+As a rule of thumb, receiving encouraging feedback from long-standing project
+developers, and particularly members of the relevant [working group] is a good
+indication that the RFC is worth pursuing.
+
+
+## The RFC life-cycle
+[The RFC life-cycle]: #the-rfc-life-cycle
+
+In short, to get a major feature added to IPFS, one must first get the RFC
+merged into the RFC repository as a markdown file. At that point the RFC is
+"Accepted" and may be implemented with the goal of eventual inclusion into the corresponding protocols or libraries.
+
+As it grows from an idea to shipping in a supported release, an SDK RFC will traverse along:
+
+1. **Draft**: The owner of the SDK RFC has started to draft up how the subject will be handled and may be reviewing with a core group.  Comments are certainly welcome at this stage even though the owner hasn't worked through enough details to ask for...
+2. **Review**: This SDK RFC is in a review period. Stakeholders and the owner may still be iterating on some final details before signoff. A minimum review period has been defined.
+3. **Final Call**
+4. **Accepted**: All stakeholders have signed off and this SDK RFC is now or will be implemented soon.
+
+
+### Status: Draft
+
+When you've done the preparation described in [Before creating an RFC] and are ready to submit your RFC, create the pull request following these steps:
+
+
+  - Fork the RFC repo [RFC repository]
+  - Copy `0000-template-rfc.md` to `rfcs/0000-my-feature.md` (where "my-feature" is
+    descriptive. don't assign an RFC number yet).
+  - Fill in the RFC. Put care into the details: RFCs that do not present
+    convincing motivation, demonstrate understanding of the impact of the
+    design, or are disingenuous about the drawbacks or alternatives tend to be
+    poorly-received.
+  - Submit a pull request. As a pull request the RFC will receive design
+    feedback from the larger community, and the author should be prepared to
+    revise it in response.
```

Comment:
"the author should be prepared to respond to feedback and possibly revise the proposal" sounds like slightly better wording to me.

Reply:
✅  applied.